hack/in-container.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -ex
+
+source hack/config.sh
+source hack/common.sh
+
+# Execute the build
+[ -t 1 ] && USE_TTY="-it"
+$KUBEVIRT_CRI run ${USE_TTY} \
+    --rm \
+    -v ${SSP_DIR}:${WORK_DIR}:rw,Z \
+    -e RUN_UID=$(id -u) \
+    -e RUN_GID=$(id -g) \
+    -e GOCACHE=/gocache \
+    -w ${WORK_DIR} \
+    ${TESTS_BUILD_TAG} "$1"
```

Comment:
If running `make build-functests`, this evaluates to `docker run ... ./hack/build-functests.sh`, but I dont see `hack/build-functests.sh` copied to the container in the Dockerfile, am I missing anything?

Reply:
`-v ${SSP_DIR}:${WORK_DIR}:rw,Z \` flag would pass bind the `SSP_DIR` path into the container and `-w` would make it the working directory.
`$1` at the end would be the argument to run inside the container which has the `SSP_DIR` already mounted; `-it` flag is already passed.
So it would be just `./hack/build-functests.sh` inside the container

Reply:
Ah, I missed that, makes sense